### PR TITLE
docs: link from source to source

### PR DESCRIPTION
**Pull request by zbwright**

### CODE REVIEW

**Comment by zbwright** [comment]
@athai suggests these might want to be hard links, rather than relative, as they're linking across repos. What say ye, @crawford ? This will be the first time we've done this in the docs, so let's figure it out, and I'll fix other instances as I find them.

**Comment by crawford** [comment]
They just need to be GitHub URLs (as they were written before).

File: Documentation/alm/prometheus-ocs.md [hunk 1]
```diff
@@ -55,11 +55,11 @@ Then, configure Prometheus to discover these targets.
 
 For more information, see [Application Monitoring][app-monitoring].
 
-
+
 [app-monitoring]: https://coreos.com/tectonic/docs/latest/tectonic-prometheus-operator/user-guides/application-monitoring.html
```
Comment (crawford) [inline Documentation/alm/prometheus-ocs.md]: Can you fix this link while you are at it?

File: Documentation/alm/prometheus-ocs.md [hunk 2]
```diff
@@ -55,11 +55,11 @@ Then, configure Prometheus to discover these targets.
...
-

+
 [app-monitoring]: https://coreos.com/tectonic/docs/latest/tectonic-prometheus-operator/user-guides/application-monitoring.html
 [client-library]: https://prometheus.io/docs/instrumenting/clientlibs/
-[exposing-prometheus]:  https://coreos.com/operators/prometheus/docs/latest/user-guides/exposing-prometheus-and-alertmanager.html
-[alerting-md]:  https://coreos.com/operators/prometheus/docs/latest/user-guides/alerting.html
+[exposing-prometheus]:  https://github.com/coreos/prometheus-operator/blob/master/Documentation/user-guides/exposing-prometheus-and-alertmanager.md
+[alerting-md]: https://github.com/coreos/prometheus-operator/blob/master/Documentation/user-guides/alerting.md
 [monitoring-apps]: https://coreos.com/tectonic/docs/latest/tectonic-prometheus-operator/user-guides/application-monitoring.html
```
Comment (crawford) [inline Documentation/alm/prometheus-ocs.md]: This one too.

Comment (zbwright) [reply Documentation/alm/prometheus-ocs.md]: private repo.

**Pull request merged**